src/import/FSHImporter.ts

```diff
@@ -210,6 +212,8 @@ export class FSHImporter extends FSHVisitor {
       this.currentFile = null;
     });
 
+    this.expandContextPaths(docs);
```

Comment:
We don't need to keep them, but it won't matter once I refactor to use the approach you detailed above.

Reply:
I'm not 100% sure my approach is better.  There may be some wrinkles in the plan... so I guess we will see!

Reply:
I've started to work on using your approach, and the one thing I am finding tricky is the question of where to apply context paths "on the fly". I was thinking within `visitPath` should work, but that isn't as clean as I'd like, because sometimes we don't `visitPath`, and sometimes we `visitPaths` since there are multiple. Did you have an idea of where that path resolving would go?

By the way, this also reminds me to add a test for multiple paths. The way I had imagined multiple paths working is:
```
Profile: Foo
Parent: Patient
* name
  * family, given MS
```
=
```
Profile: Foo
Parent: Patient
* name.family MS
* name.given MS
```

Reply:
Like I said, I hadn't fully thought it out. ;-)  It seems like `visitPath`/`visitPaths` would make sense.  In what cases do we _not_ visit the path?  Can we update those so they _do_ `visitPath`?  Another possible place would be in the `visitXYZRules` functions (e.g., `visitSDRules`, `visitInstanceRules`, etc) -- which would look at the spacing to update the path context; but then you still need to apply that context in `visitPath`/`visitPaths` (and any other places path is involved that doesn't visit the path).

As for multi-path rules, I think what you suggest makes sense -- but the other case is that _this_ can't work:
```
* name, active MS
  * given MS
```

Reply:
BTW -- if it turns out that the approach I suggested is more prone to error (you need to "remember" to handle the path context in a bunch of places, and future code updates might "forget" to do it), then that might be a good reason to stick to your original implementation.

Reply:
Oh yeah good point about the multi-path rules being followed by an indent. I will add that as well.

Sometimes we don't `visitPath` because there is no path to be visited, like a `CaretRule` may or may not have `.path()` defined. I'm still not sure which approach is going to be better. I haven't had any time to try it as you suggested. I like the idea of `PathRules` never even being added to the `rules` array of an entity, since they really don't have to be in theory.

Reply:
> As for multi-path rules, I think what you suggest makes sense -- but the other case is that _this_ can't work:
> 
> ```
> * name, active MS
>   * given MS
> ```

@cmoesel and I discussed this, and decided it's probably fine if this just works like 
```
* active, name MS
  * given MS
```
=
```
* active, name MS
* name.given MS
```
So the last thing in a multi-path rule sets the following context.